web/client/epics/login.js

```diff
@@ -58,6 +58,20 @@ const reloadMapConfig = (action$, store) =>
         return Rx.Observable.of(configureError(e));
     });
 
+const reloadNewMapConfig = (action$, store) =>
```

Comment:
I suggest to do everything in hte reloadMapConfig epic

Reply:
@offtherailz I am not sure if you have noticed that the `reloadMapConfig` filter out the `LOGOUT` action while the required behaviour need to work on `LOGOUT` action and specifically the logout that happens on new map context(not all logout). I found out that modifying `reloadMapConfig` to include `LOGOUT` action and the required behaviour was difficult and complicated and I couldn't achieve it as the result I introduced a new epic to tackle only the new behaviour. That was clean and readable than the other way around but I am open if there is a way I could incorporate the behaviour on existing epic, if you can get sometime then we can talk about this challenge